### ISSUE TITLE
Commandable steals my exit codes

### ISSUE THREAD

**Issue by AndrewVos** [mikbe/commandable#1]
When calling Kernel.exit with an exit code I expect that exit code to be output.

The offending line of code is here:

https://github.com/mikbe/commandable/blob/master/lib/commandable/commandable.rb#L157

Example code:

``` ruby
require 'commandable'

class Example
  extend Commandable

  command "example command"
  def command(meh)
    if meh == "hello"
      Kernel.exit(1)
    end
  end
end

Commandable.execute(ARGV)
```

Notice at the end when I echo out the exit code it's zero.

``` bash
~/Desktop $ ruby example.rb command hello

 Bleep, bloop, bleep! Danger Will Robinson! Danger!

 Error: #<SystemExit: exit>

 Backtrace:
 example.rb:9:in `exit'
 example.rb:9:in `command'
 /Users/andrewvos/.rvm/gems/ruby-1.9.2-p290/gems/commandable-0.2.1/lib/commandable/commandable.rb:240:in `block (2 levels) in execution_queue'
 /Users/andrewvos/.rvm/gems/ruby-1.9.2-p290/gems/commandable-0.2.1/lib/commandable/commandable.rb:151:in `call'
 /Users/andrewvos/.rvm/gems/ruby-1.9.2-p290/gems/commandable-0.2.1/lib/commandable/commandable.rb:151:in `block in execute'
 /Users/andrewvos/.rvm/gems/ruby-1.9.2-p290/gems/commandable-0.2.1/lib/commandable/commandable.rb:150:in `each'
 /Users/andrewvos/.rvm/gems/ruby-1.9.2-p290/gems/commandable-0.2.1/lib/commandable/commandable.rb:150:in `execute'
 example.rb:14:in `<main>'

~/Desktop $ echo $?
0
```

**Comment by mikbe** [comment]
Fixed and released in Commandable 0.2.2.

Thanks for the bug report.

**Comment by AndrewVos** [comment]
I was actually going to take a look at this today and try fix it. Can't believe you fixed it so fast! Thanks!
